src/uid2Sdk.ts

```diff
@@ -26,11 +26,20 @@ export class UID2 {
     }
     static IdentityStatus = IdentityStatus;
     static EventType = EventType;
-    static setupGoogleTag = setupGoogleTag;
+    static setupGoogleTag() {
```

Comment:
Should this be renamed to something like `setupGoogleSecureSignals`?

Reply:
This function exists in old sdks, it was updated for backward compatibility, I think we'd better not rename it.

Reply:
Maybe it's a good opportunity to provide a new name and also support the old one - that sets us on the path to eventually deprecate the old name.